github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits.yaml

```diff
@@ -15,19 +15,32 @@ presubmits:
       cluster: ibm-prow-jobs
       max_concurrency: 6
       labels:
-        preset-dind-enabled: "true"
+        preset-podman-in-container-enabled: "true"
         preset-docker-mirror-proxy: "true"
         preset-shared-images: "true"
       spec:
         containers:
-          - image: quay.io/kubevirtci/kubevirt-infra-bootstrap:v20220425-b167bb8
+          - image: quay.io/kubevirtci/bootstrap:v20220816-91d8af2
```

Comment:
@brianmcarey 
I remember you updated `quay.io/kubevirtci/kubevirt-infra-bootstrap` to  `quay.io/kubevirtci/bootstrap` right ?
here we still see the old ones before this PR changes, did you revert it ? or it is an additional place?

Reply:
it hasn't been merged yet but the rehearsals seemed to work fine with the regular bootstrap image - https://github.com/kubevirt/project-infra/pull/2226

Reply:
thanks, maybe lets merge it ?